ci/integration-tests.sh

```diff
@@ -14,16 +14,21 @@ cargo install --force --debug --path .
 echo "Running integration test for crate ${INTEGRATION}"
 
 git clone --depth=1 "https://github.com/${INTEGRATION}.git" checkout
-cd checkout || exit 1
+cd checkout
 
-# run clippy on a project, try to be verbose and trigger as many warnings as possible for greater coverage
+# run clippy on a project, try to be verbose and trigger as many warnings
+# as possible for greater coverage
 RUST_BACKTRACE=full \
 cargo clippy \
     --all-targets \
     --all-features \
-    -- --cap-lints warn -W clippy::pedantic -W clippy::nursery \
-    2>& 1 \
-| tee clippy_output
+    -- \
+    --cap-lints warn \
```

Comment:
We have `--cap-lints warn` enabled, so the only time this should return an error code != 0 is, when there is an ICE or some other unexpected error. So maybe we can use this to prevent something like this #4719 from happening?

Reply:
I think more about it. Clippy (or rustc) could fail to build the integration tests.
We don't want to check for that, do we?

Reply:
This fail on:
* stdsimd: https://travis-ci.com/rust-lang/rust-clippy/jobs/249810547#L21999
* chalk: https://travis-ci.com/rust-lang/rust-clippy/jobs/249810540
* rls: https://travis-ci.com/rust-lang/rust-clippy/jobs/249810541